.github/workflows/tests.yml

```diff
@@ -0,0 +1,82 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - unicorn
+    tags:
+      - '!*.*.*'
+  pull_request:
+    branches:
+      - unicorn
+
+jobs:
+
+  backend:
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    runs-on: ubuntu-latest
+    name: Python ${{ matrix.python-version }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install and set up Poetry
+      run: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+    - name: Install dependencies
+      run: |
+        source $HOME/.poetry/env
+        poetry install -vvv
```

Comment:
It's better to do this during poetry setup:

https://github.com/chaoss/grimoirelab-perceval/blob/master/.github/workflows/tests.yml#L30